packages/editor/src/store/test/reducer.js

```diff
@@ -2263,4 +2264,53 @@ describe( 'state', () => {
 			} );
 		} );
 	} );
+
+	describe( 'postSavingLock', () => {
+		it( 'returns empty object by default', () => {
+			const state = postSavingLock( undefined, {} );
+
+			expect( state ).toEqual( {} );
+		} );
+
+		it( 'returns correct post locks when locks added and removed', () => {
+			let state = postSavingLock( undefined, {
+				type: 'LOCK_POST_SAVING',
+				lockName: 'test-lock',
+			} );
+
+			expect( state ).toEqual( {
+				'test-lock': true,
+			} );
+
+			state = postSavingLock( state, {
```

Comment:
Minor: It's generally a good practice in reducer tests to apply `deepFreeze` to the `state` passed to the reducer. It will indirectly protect against unintentional mutations (via thrown errors failing the tests). As written here, technically this test would be satisfied by the following theoretical misbehaving reducer:

```js
export function postSavingLock( state = {}, action ) {
	switch ( action.type ) {
		case 'LOCK_POST_SAVING':
		case 'UNLOCK_POST_SAVING':
			state[ action.lockName ] = action.type === 'LOCK_POST_SAVING';
			break;

	return state;
}
```

Reply:
added in https://github.com/WordPress/gutenberg/pull/10649/commits/0696c0c9288ba5b8c9994a6c3b09be79abc4d65f